config/app.config

```diff
@@ -53,6 +53,12 @@ lsst.dataAccess.db = ''
 ehcache.multicast.port = 4016
 visualize.fits.Security= true
 
+// IRSA Periodogram API
+irsa.gator.service.periodogram.url = "http://bacchus.ipac.caltech.edu:9027/cgi-bin/periodogram/nph-periodogram_api"
+
+// Periodogram API request parameter list definition, separated by space
+irsa.gator.service.periodogram.keys = "x y alg step_method pmin pmax step_size peaks"
```

Comment:
Well, imagine that the dev/test API has different keys than OPS, i still want to control that.